GPTsettings.py

```diff
@@ -33,11 +32,11 @@ class GPTsettings:
                                         # DONT USE NONE 
 
         'body': """You should consider refactoring this piece of code, for example you could try:
-        ```suggestion
-        def a_func(a:int, b:int):
-            # func content
-            return # something
-        ```
+```suggestion
+def a_func(a:int, b:int):
+    # func content
+    return # something
+```
```

Comment:
Let's not play the indentation limbo, huh? Keep your markdown code blocks cleanly indented. This one's trying to escape the fence like a dog after a squirrel. Keep it consistent!